snippet-template.md

```diff
@@ -11,4 +11,4 @@ const functionName = arguments =>
 
 ```js
 functionName('sampleInput'); // 'sampleOutput'
-```
+```
```

Comment:
Apparently the last line changed... I don't see anything different, but I don't care about this at all.